armory/eval/evaluator.py

```diff
@@ -84,20 +84,28 @@ def ensure_image_present(self, image_name: str) -> str:
         # look first for  the versioned and then the unversioned, return if hit
         # if there is a tag present, use that. otherwise add the current version
         if ":" in image_name:
-            check = image_name
+            checks = (image_name,)
         else:
+            # TODO: This needs to be fixed if 'image_name' does not refer to twosixarmory image
+            #   There should be a more explicit check for specific armory image names.
             check = f"{image_name}:{armory.__version__}"
+            check_previous = ".".join(check.split(".")[:3])
```

Comment:
In some discussion we considered this order
1. major.minor.patch.hash
2. major.minor.patch
3. major.minor

To cut down on docker builds on every commit. This code implements (1) and (2) but not (3). I just wanted to check that omission was intentional. If we find we want (3) we can always add it later.

Reply:
I omitted it because we haven't published any images with major.minor